src/Vehicle/FTPManager.cc

```diff
@@ -237,6 +324,44 @@ void FTPManager::_openFileROAckOrNak(const MavlinkFTP::Request* ackOrNak)
     }
 }
 
+void FTPManager::_createFileROAckOrNak(const MavlinkFTP::Request* ackOrNak)
```

Comment:
* typo: NACK 
Could be better to rename to createFileROResponse?

Reply:
Naming is done keeping in mind the standard set by QGC developers